sdk/communication/azure-communication-administration/azure/communication/administration/_communication_identity_client.py

```diff
@@ -152,4 +153,16 @@ def revoke_tokens(
             user.identifier if user else None,
             tokens_valid_from=issued_before,
             **kwargs)
- 
\ No newline at end of file
+    
+    def get_authentication_policy(self, endpoint, credential):
```

Comment:
it seems that this method is identical with the async version. Instead of copying I would recommend, creating `_utils.py` file and add that method there

Reply:
Actually, it is a better idea to add this to the `_shared/utils` as it seems other packages might use that as well?

Reply:
Solved it!